problems/sumNumbersAtEvenIndices.js

```diff
@@ -12,7 +12,14 @@
 * returns: 45
 */
 function sumNumbersAtEvenIndices() {
-
+    let arr = [];
+    for(let i = 0; i < arr.length; i++) {
```

Comment:
you forget the parameter again and`arr.length` here is zero. The loop looks great, we could do without the else statement since it's not needed